hack/e2e/environment.sh

```diff
@@ -30,6 +30,9 @@ export CONTROL_PLANE_MACHINE_COUNT=${CONTROL_PLANE_MACHINE_COUNT:-3}
 # shellcheck disable=SC2016
 export WORKER_MACHINE_COUNT=${WORKER_MACHINE_COUNT:-1}
 
+export NUM_OF_MASTER_REPLICAS=${CONTROL_PLANE_MACHINE_COUNT}
```

Comment:
Can please add a comment to explain why we are naming the same this differently `CONTROL_PLANE_MACHINE_COUNT`  and `NUM_OF_MASTER_REPLICAS`

Reply:
Can you please answer the question asked above?

Reply:
I added the comments to explain the question in the code, not sure why it is not showed here, but you can check in `file changed`, and I can also add my answer here: 
# These two variables below are required to render the cluster template dynamically from the metal3-dev-env.
# This template is used by default, but users can change to use the static templates by changing the flavor.